### PR TITLE
chore: update release verion

### PR DESCRIPTION
**Pull request by wuwentao**
# PR Description

1. add code owner
2. update release verion in mainfest.json

### CODE REVIEW

**Comment by chemelli74** [comment]
I would wait a little bit, just to fix the last single issue in #99 and then we can release the new version

**Comment by wuwentao** [comment]
> I would wait a little bit, just to fix the last single issue in #99 and then we can release the new version

yes, will wait for #99 done, please don't worry.
I have mark this PR as draft.

**Review by rokam — APPROVED** [review]

**Comment by rokam** [comment]
> > I would wait a little bit, just to fix the last single issue in #99 and then we can release the new version
> 
> yes, will wait for #99 done, please don't worry. I have mark this PR as draft.

We can merge. Just don´t  create a new release

File: custom_components/midea_ac_lan/__init__.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     model: str = config_entry.data[CONF_MODEL]
     subtype = config_entry.data.get(CONF_SUBTYPE, 0)
     protocol: int = config_entry.data[CONF_PROTOCOL]
-    customize: str = config_entry.options[CONF_CUSTOMIZE]
+    customize: str = config_entry.options.get(CONF_CUSTOMIZE, "")
```
Comment (chemelli74) [inline custom_components/midea_ac_lan/__init__.py:191]: We should do a separate PR to fix bugs

**Review by chemelli74 — DISMISSED** [review]

File: custom_components/midea_ac_lan/__init__.py (hunk 1 above)
Comment (Necroneco) [reply custom_components/midea_ac_lan/__init__.py:191]: Agree

Comment (wuwentao) [reply custom_components/midea_ac_lan/__init__.py:191]: done, via https://github.com/wuwentao/midea_ac_lan/pull/115
